### PR TITLE
Show user prefered timezone in timetable balloon

### PR DESCRIPTION
**Pull request by DavidAndreev**
Don't take this too seriously. I'm not sure if I'm doing it right cause I don't know what that DisplayTZ does, but it seams to fix the problem. 
works
But when I try in management timetable it shows the right time but the time rows of timetable are not right. but maybe thats that thing that Pedro was fixing.
doesnt work

### CODE REVIEW

**Comment by ThiefMaster** [comment]
Don't use `DisplayTZ` directly - there's ` display_tzinfo` property on `Event` that wraps it nicely.

**Comment by ThiefMaster** [comment]
Also, in management we always want the event's timezone - only in display areas we want the "display timezone" (that takes the user's timezone preference into account)

**Comment by pferreir** [comment]
Maybe this should include my related PR? Feel free to do it as you guys
wish.

**Comment by DavidAndreev** [comment]
So is there a way to check if we are in management from within the template and based on that I will show the proper time zone ?

**Comment by ThiefMaster** [comment]
Check when we pass `editable=true` to the macro. That might be enough to check.

File: indico/modules/events/timetable/util.py [hunk 1]
```diff
@@ -19,11 +19,13 @@
 from collections import defaultdict
 from operator import attrgetter
 
+from babel.dates import get_timezone
 from flask import render_template, session
 from pytz import utc
 from sqlalchemy import Date, cast
 from sqlalchemy.orm import joinedload, subqueryload, undefer, contains_eager
 
+from MaKaC.common.timezoneUtils import DisplayTZ
```
Comment (ThiefMaster) [inline indico/modules/events/timetable/util.py]: unused

File: indico/modules/events/timetable/util.py [hunk 2]
```diff
@@ -19,11 +19,13 @@
 from collections import defaultdict
 from operator import attrgetter
 
+from babel.dates import get_timezone
```
Comment (ThiefMaster) [inline indico/modules/events/timetable/util.py]: unused

**Comment by ThiefMaster** [comment]
LGTM, but just like #2730 I'd rather merge this into v1.9.8 - at least if we want to hotfix it in prod.

**Comment by ThiefMaster** [comment]
Added to #2730

**Pull request closed without merging**